code/modules/reagents/Chemistry-Goon-Toxins.dm

```diff
@@ -204,4 +204,18 @@ datum/reagent/itching_powder/on_mob_life(var/mob/living/M as mob)
 	result = "facid"
 	required_reagents = list("sacid" = 1, "fluorine" = 1, "hydrogen" = 1, "potassium" = 1)
 	result_amount = 4
-	required_temp = 380
\ No newline at end of file
+	required_temp = 380
+
+datum/reagent/paprika
+	name = "Paprika"
+	id = "paprika"
+	description = "A spice made from chili peppers."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+
+datum/reagent/paprika/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(prob(50))
+		M.adjustFireLoss(1)
```

Comment:
Please no, don't add fire damage to it if you add it to chili pepper, we don't need more harmful recipes. Plus, capsaicin already exists.